src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,52 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api/api';
 import './CurrentUser.scss';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  userId: number,
+  selectedUser: (userId: number) => void,
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+export const CurrentUser: React.FC<Props> = ({ userId, selectedUser }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [loading, setLoading] = useState(false);
+  const [loadingError, setLoadingError] = useState(false);
+
+  useEffect(() => {
+    getUser(userId)
+      .then(response => setUser(response))
+      .then(() => setLoading(true))
+      .catch(() => setLoadingError(true));
+  }, [userId]);
+
+  return (
+    <>
+      {!loading && (
+        <p>Please wait, loading</p>
+      )}
+
+      {(loading && loadingError) && (
+        <p>Failed to load</p>
+      )}
+
+      {(loading && !loadingError) && (
+        <div className="CurrentUser">
+          <h2 className="CurrentUser__title">
+            <span>
+              {`Current user: ${user?.id}`}
```

Comment:
check if user exists before rendering, because now if user is null, here will be empty layout